### PR TITLE
Offer a provider to load script from the chaosapi

### LINKED ISSUE #5: Offer provider to load script from chaosapi

### PR DESCRIPTION
**Pull request by bugraderre**
Besides the mentioned feature this commit contains further improvements:
* Cleaned up the README.md
* Improved DRY principles with the __load_settings() function

Resolves: #5

### CODE REVIEW

File: pdchaoskit/provider.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from chaoslib.settings import get_loaded_settings
+from logzero import logger
+
+from pdchaoskit.api.scripts import get_script
+from pdchaoskit.api.session import client_session
+
+
+def script_content(id: str) -> str:
```
Comment (mkaszub) [inline pdchaoskit/provider.py]: What if we call this module "scripts" and and this method get_content() or get_script_content().

So very similar what you did in api module but this one is already handling you the api call

Provider may be a bit too generic. Checkout how other modules are called in chaostoolkit.

**Review by mkaszub — COMMENTED** [review]
I like it. Just on comment about naming of the module. Good job

File: pdchaoskit/provider.py (hunk 1 above)
Comment (bugraderre) [reply pdchaoskit/provider.py]: Good idea. Done

**Review by mkaszub — APPROVED** [review]
Nice!

**Pull request merged**